pkg/controller/plan/adapter/openstack/builder_test.go

```diff
@@ -0,0 +1,19 @@
+package openstack
+
+import (
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
```

Comment:
Looks like you are using ginkgo v1 and v2 in this repo? When starting from scratch I would consider using v2.

Reply:
yeah, we recently added v2 without removing v1 - we should do that (separately)